Modules/Loadable/VolumeRendering/Widgets/qSlicerCPURayCastVolumeRenderingPropertiesWidget.cxx

```diff
@@ -73,11 +73,11 @@ ::populateRenderingTechniqueComboBox()
 {
   this->RenderingTechniqueComboBox->clear();
   this->RenderingTechniqueComboBox->addItem(
-    "Composite With Shading", vtkMRMLViewNode::Composite);
+    tr("Composite With Shading"), vtkMRMLViewNode::Composite);
```

Comment:
Methods in private implementation classes (class names ending with `...Private`) do not have `tr()` function, but the `tr()` function of the public class must be used - see https://github.com/Slicer/SlicerLanguagePacks/blob/main/DevelopersManual.md#how-to-use-tr--in-non-qobject-classes
I don't comment on all the other similar cases like this, but please fix them all.
```suggestion
    qSlicerCPURayCastVolumeRenderingPropertiesWidget::tr("Composite With Shading"), vtkMRMLViewNode::Composite);
```